src/app/reference/ci-workflows/recording-strategies/page.md

```diff
@@ -3,42 +3,14 @@ title: Recording strategies
 description: Depending on what problem you’re trying to solve, there are two primary ways of using Replay.
 ---
 
-## Record failures on PRs
+## Record on PRs
 
-This is useful for debugging failing tests in pull requests. It only records failing tests so there’s minimal overhead, and those replays will be available for debugging immediately after tests finish.
-
-Run your tests and pass the mode `record-on-retry` .
-
-{% tabs labels=["npm", "yarn", "pnpm"] %}
-{% tab %}
-
-```sh
-npx @replayio/cypress run --mode record-on-retry
-```
-
-{% /tab %}
-{% tab %}
-
-```sh
-yarn @replayio/cypress run --mode record-on-retry
-```
-
-{% /tab %}
-{% tab %}
-
-```sh
-pnpx @replayio/cypress run --mode record-on-retry
-```
-
-{% /tab %}
-{% /tabs %}
+This is useful for debugging failing tests in pull requests. It records tests and those replays will be available for debugging immediately after tests finish.
```

Comment:
I can't figure out how to actually view this content. When I try to open this page by URL or search, I get redirected to `reference/ci-workflows/upload-strategies`